rust/tests/loadtest/src/main.rs

```diff
@@ -118,6 +160,33 @@ async fn main() -> Result<(), GooseError> {
     Ok(())
 }
 
+/// Initializes logging with optional Windows Event Log support.
+fn init_logging() {
+    #[cfg(windows)]
+    {
+        use tracing_subscriber::layer::SubscriberExt as _;
+
+        match logging::windows_event_log::layer(EVENT_LOG_SOURCE) {
+            Ok(layer) => {
+                tracing_subscriber::registry().with(layer).init();
+            }
+            Err(e) => {
+                eprintln!(
+                    "Warning: Could not initialize Windows Event Log: {e}\n\
+                    Events will not be logged to Event Viewer.\n\
+                    Set EVENTLOG_DIRECTIVES to control Event Log filtering (default: info)."
+                );
+                tracing_subscriber::registry().init();
+            }
+        }
```

Comment:
Should we make `windows_event_log` return a dummy layer on non-windows platforms to make this conditional code go away?